### PR TITLE
Connexion header

### PR DESCRIPTION
**Pull request by Gauthier-Lhomme**
Ternary and route

### CODE REVIEW

**Review by Thoms-C — CHANGES_REQUESTED** [review]
console.log =>line 21 => UpdateCLient Page

**Review by Thoms-C — DISMISSED** [review]
Noice !

File: src/components/ConnexionLogin.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    axios
+      .post("http://localhost:3001/auth/login", values)
+      .then((res) => {
+        setToken(res.data);
+        history.push("/clientpage");
+      })
+      .catch((e) => {
+        console.warn(e);
```
Comment (LoicBrassart) [inline src/components/ConnexionLogin.jsx]: %ùµ£#~% !!!

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Nope, won't read the same effing EIGHT HUNDREDS lines over and over.
Get this repo in order, I'll resume proofreading asap

**Review by Thoms-C — APPROVED** [review]
Console.warn deleted ok !

File: src/components/ConnexionLogin.jsx [hunk 2]
```diff
@@ -17,15 +17,10 @@ export default function useForm({ setToken }) {
...
-    axios
-      .post("http://localhost:3001/auth/login", values)
-      .then((res) => {
-        setToken(res.data);
-        history.push("/clientpage");
-      })
-      .catch((e) => {
-        console.warn(e);
-      });
+    axios.post("http://localhost:3001/auth/login", values).then((res) => {
```
Comment (LoicBrassart) [inline src/components/ConnexionLogin.jsx:20]: You'll need to variabilize this if you ever want it to work in production

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**